modules/serverless_negs_with_endpoint/main.tf

```diff
@@ -0,0 +1,320 @@
+        }
+      }
+
+      dynamic "cache_key_policy" {
+        for_each = each.value.cdn_policy.cache_key_policy != null ? [1] : []
+        content {
+          include_host           = each.value.cdn_policy.cache_key_policy.include_host
+          include_protocol       = each.value.cdn_policy.cache_key_policy.include_protocol
+          include_query_string   = each.value.cdn_policy.cache_key_policy.include_query_string
+          query_string_blacklist = each.value.cdn_policy.cache_key_policy.query_string_blacklist
+          query_string_whitelist = each.value.cdn_policy.cache_key_policy.query_string_whitelist
+          include_http_headers   = each.value.cdn_policy.cache_key_policy.include_http_headers
+          include_named_cookies  = each.value.cdn_policy.cache_key_policy.include_named_cookies
+        }
+      }
+
+      dynamic "bypass_cache_on_request_headers" {
+        for_each = toset(each.value.cdn_policy.bypass_cache_on_request_headers) != null ? each.value.cdn_policy.bypass_cache_on_request_headers : []
+        content {
+          header_name = bypass_cache_on_request_headers.value
+        }
+      }
+    }
+  }
+
+  dynamic "outlier_detection" {
+    for_each = each.value.outlier_detection != null && (var.load_balancing_scheme == "INTERNAL_SELF_MANAGED" || var.load_balancing_scheme == "EXTERNAL_MANAGED") ? [1] : []
+    content {
+      consecutive_errors                    = each.value.outlier_detection.consecutive_errors
+      consecutive_gateway_failure           = each.value.outlier_detection.consecutive_gateway_failure
+      enforcing_consecutive_errors          = each.value.outlier_detection.enforcing_consecutive_errors
+      enforcing_consecutive_gateway_failure = each.value.outlier_detection.enforcing_consecutive_gateway_failure
+      enforcing_success_rate                = each.value.outlier_detection.enforcing_success_rate
+      max_ejection_percent                  = each.value.outlier_detection.max_ejection_percent
+      success_rate_minimum_hosts            = each.value.outlier_detection.success_rate_minimum_hosts
+      success_rate_request_volume           = each.value.outlier_detection.success_rate_request_volume
+      success_rate_stdev_factor             = each.value.outlier_detection.success_rate_stdev_factor
+
+      dynamic "base_ejection_time" {
+        for_each = each.value.outlier_detection.base_ejection_time != null ? [1] : []
+        content {
+          seconds = each.value.outlier_detection.base_ejection_time.seconds
+          nanos   = each.value.outlier_detection.base_ejection_time.nanos
+        }
+      }
+
+      dynamic "interval" {
+        for_each = each.value.outlier_detection.interval != null ? [1] : []
+        content {
+          seconds = each.value.outlier_detection.interval.seconds
+          nanos   = each.value.outlier_detection.interval.nanos
+        }
+      }
+    }
+  }
+}
+
+resource "google_compute_region_network_endpoint_group" "serverless_neg" {
```

Comment:
@A0G1 where r u passing this NEG as backend?

Reply:
The neg ID is passed through the module definition [link](https://github.com/terraform-google-modules/terraform-google-lb-http/pull/437/files#diff-c4fc88f2eb00b0395e6cd581ce3132302177d2a268a45ff16ced75a8742318f3R47) which we get from outputs.tf [link](https://github.com/terraform-google-modules/terraform-google-lb-http/pull/437/files#diff-de8a2d4c7ff11fdc390da4fb09066ac3fe30f29703309bdc74602b38d0486352R58).